vllm/v1/attention/backends/triton_attn.py

```diff
@@ -74,6 +74,7 @@ class LocalAttentionMetadata:
 class TritonAttentionMetadataBuilder(
         AttentionMetadataBuilder[TritonAttentionMetadata]):
     full_cudagraph_supported: ClassVar[bool] = True
+    force_separate_routine: ClassVar[Optional[bool]] = None
```

Comment:
This can be set to None by default on the `AttentionMetadataBuilder` abstract class